ci/recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ requirements:
     - cutadapt
     - pigz
     - pandas
+    - numpy
```

Comment:
Numpy is a secondary dependency via pandas, so this doesn't change the overall dependency chain for this plugin.